inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -288,7 +288,7 @@ CREATE TABLE IF NOT EXISTS `inlong_stream_field`
     `inlong_group_id`     varchar(256) NOT NULL COMMENT 'Owning inlong group id',
     `inlong_stream_id`    varchar(256) NOT NULL COMMENT 'Owning inlong stream id',
     `is_predefined_field` tinyint(1)   DEFAULT '0' COMMENT 'Whether it is a predefined field, 0: no, 1: yes',
-    `field_name`          varchar(20)  NOT NULL COMMENT 'field name',
+    `field_name`          varchar(120)  NOT NULL COMMENT 'field name',
```

Comment:
Please change the same SQL file under the manager-test, thx.

Reply:
done